saleor/account/migrations/0081_update_user_is_confirmed.py

```diff
@@ -0,0 +1,59 @@
+from django.db import migrations, transaction
+from django.db.models import QuerySet
+from ..models import User
+
+
+BATCH_SIZE = 5000
```

Comment:
Please do not use magic numbers. Could you please measure ram usage and add comment about in the code?